project-base/tests/FrontendApiBundle/Test/GraphQlTestCase.php

```diff
@@ -205,4 +235,124 @@ protected function assertResponseContainsArrayOfExtensionValidationErrors(array
         $this->assertArrayHasKey('validation', $errors[0]['extensions']);
         $this->assertIsArray($errors[0]['extensions']['validation']);
     }
+
+    /**
+     * @param array $expectedOrderItems
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Price
+     */
+    protected function getOrderTotalPriceByExpectedOrderItems(array $expectedOrderItems): Price
+    {
+        $totalPriceWithVat = Money::zero();
+        $totalPriceWithoutVat = Money::zero();
+
+        foreach ($expectedOrderItems as $expectedOrderItem) {
+            $expectedOrderItemTotalPrice = $expectedOrderItem['totalPrice'];
+            $totalPriceWithVat = $totalPriceWithVat->add(Money::create($expectedOrderItemTotalPrice['priceWithVat']));
+            $totalPriceWithoutVat = $totalPriceWithoutVat->add(
+                Money::create($expectedOrderItemTotalPrice['priceWithoutVat'])
+            );
+        }
+        return new Price($totalPriceWithoutVat, $totalPriceWithVat);
+    }
+
+    /**
+     * @param string $vatRate
+     * @return \Shopsys\FrameworkBundle\Model\Pricing\Vat\Vat
+     */
+    protected function getVatByVatRate(string $vatRate): Vat
```

Comment:
I would like to make this private as it's used only in the same file

Reply:
This method has been removed, it is not necessary any more.